magento2/usr/local/share/container/baseimage-30.sh

```diff
@@ -9,6 +9,12 @@ if [ "$IMAGE_VERSION" -ge 2 ]; then
     PRODUCTION_ENVIRONMENT="$BUILD_PRODUCTION_ENVIRONMENT" MAGENTO_MODE="$BUILD_MAGENTO_MODE" DEVELOPMENT_MODE="$BUILD_DEVELOPMENT_MODE" do_magento2_build
   }
 
+  alias_function do_start do_magento2_start_inner
+  do_start() {
+    do_magento2_start_inner
+    do_composer_config
```

Comment:
How does M2 use this in non dev mode scenarios?

Reply:
Sorry I see from your comment above. I guess as we set do_development_start to only run if RUN_BUILD=true the previously more logical place was gone